src/hooks/useConcepts.tsx

```diff
@@ -0,0 +1,14 @@
+import useSWRImmutable from 'swr/immutable';
+import { openmrsFetch, OpenmrsResource } from '@openmrs/esm-framework';
+
+const conceptRepresentation =
+  'custom:(uuid,display,conceptMappings:(conceptReferenceTerm:(conceptSource:(name),code)))';
+
+export function useConcepts(references: Array<string>) {
+  // TODO: handle paging (ie when number of concepts greater than default limit per page)
```

Comment:
Any thoughts on this?  Is there any existing pattern/code for paging through the results if necessary?

Reply:
I'm thinking @samuelmale  we can leave this as-is for now and I can create a future ticket about figuring out how to handle forms with lots of concepts?